src/components/ResultsView.tsx

```diff
@@ -910,9 +910,37 @@ const ResultsView: typeof window.roamjs.extension.queryBuilder.ResultsView = ({
             <div>Layout `{layout}` is not supported</div>
           )
         ) : (
-          <p className="px-2 py-3 flex justify-between items-center mb-0">
-            <i>No Results Found</i>
-          </p>
+          <>
+            <p className="px-2 py-3 flex justify-between items-center mb-0">
+              <i>No Results Found</i>
+            </p>
+            <div style={{ background: "#eeeeee80"}}>
+              <div
+                className="flex justify-between items-center"
+                style={{
+                  opacity: 0.8,
+                  padding: "8px 4px",
+                }}
+              >
+                <span>
+                  <i style={{ opacity: 0.8 }}>
+                    <Button
+                      icon={showContent ? "caret-down" : "caret-right"}
+                      minimal
+                      small
+                      onClick={() => setShowContent(!showContent)}
+                      style={{
+                        marginRight: 4,
+                      }}
+                    />
+                    Showing {paginatedResults.length} of {results.length}{" "}
```

Comment:
Note that because this is the case where there's no results found, this will always be `Showing 0 of 0`, which is redundant with the message above. We could probably just have the carot or combine it with the message above.

No need to do so as part of this task, can handle in a followup if you like. merging!

Reply:
Ah, good point!

Although, as a user, there is something comforting about uniform or familiar interface components.  
Interesting tradeoffs to consider.